modules/core/core-repo/src/main/java/com/enonic/xp/repo/impl/elasticsearch/SearchRequestBuilderFactory.java

```diff
@@ -163,8 +187,22 @@ public Builder client( final Client client )
             return this;
         }
 
+        public Builder scrollTime( final TimeValue scrollTime )
+        {
+            this.scrollTime = scrollTime;
+            return this;
+        }
+
+        private void validate()
+        {
+            Preconditions.checkNotNull( query, "query must be set." );
```

Comment:
@vbradnitski I suggest to use `Objects.requireNonNull()` instead of `Preconditions.checkNotNull` because we are trying to avoid uses guava library.

Reply:
Better do it in one shot, not gradually. Leave it for now.